src/vector/index.js

```diff
@@ -32,10 +32,10 @@ require('babel-polyfill');
 // Require common CSS here; this will make webpack process it into bundle.css.
 // Our own CSS (which is themed) is imported via separate webpack entry points
 // in webpack.config.js
-require('gemini-scrollbar/gemini-scrollbar.css');
-require('gfm.css/gfm.css');
-require('highlight.js/styles/github.css');
-require('draft-js/dist/Draft.css');
+require('matrix-react-sdk/node_modules/gemini-scrollbar/gemini-scrollbar.css');
+require('matrix-react-sdk/node_modules/gfm.css/gfm.css');
+require('matrix-react-sdk/node_modules/highlight.js/styles/github.css');
+require('matrix-react-sdk/node_modules/draft-js/dist/Draft.css');
```

Comment:
I assume this didn't work without hardcoding the path? I think this might break between versions of npm.

Reply:
oh yep, I thought I'd seen that somewhere else. I don't know off hand any other way to do this, I guess other than importing the dependency directly.